### ISSUE TITLE
Regions and Territories for Accounts – can these be the same, are they different, does standard territories meet industry needs for segmentation?

### ISSUE THREAD

**Issue by crm-cat** [blackcat-/Dynamics365-Community-Schema#10]

**Comment by AdamVero** [comment]
My feeling is that territories  are usually used to reflect something "internal" - how we divide customers / prospects up in order to sell to them. This might be geographical, or based on sector/industry, or based on size (turnover, number of employees, number of locations, number of vehicles - whatever is relevant for what we sell). Sales people are associated with a Territory, and you can set up a default price list for a Territory too.
Region sounds like a property of an Account that is true from their perspective, not ours, or is simply an empirical fact. If regions are geographical, then an Account is either in region A or region B, regardless of how we sell to them.
If our territories are based on geography as well then there might be a relationship between territory and region (1:1, 1:N or N:1, depending on the model).
Region might be used for determing segmentation, language preferences, timezones or other things. It might be part of a hierarchical geography model eg Region = APAC, Country = Japan, or Country = USA, region = Mid West.
OOB CRM Territories have some limitations, especially that a manager can only be manager of one territory (you can only manage a Territory you are in, and can only be in one). Of course, a custom lookup relationship to systemuser soon gets around this for.

**Comment by scottdurow** [comment]
I agree that Region should be considered in a wider context of address validation/geocoding. Perhaps this item could be purely to do with extending Territory (the internal view as Adam puts it).
